bin/video2x.py

```diff
@@ -359,7 +368,15 @@ def absolutify_paths(config):
     elif os.path.isdir(args.input):
         # upscale videos in a directory
         Avalon.info(f'Upscaling videos in directory: {args.input}')
+
+        # determine which files are videos
+        accepted_formats = ['.mkv', '.mp4', '.webm']
```

Comment:
more formats are accepted

Reply:
I can't find a comprehensive list of formats available, I only listed the most common. `ffmpeg -formats` does not do the trick.